src/revlanguage/main.cpp

```diff
@@ -191,10 +191,13 @@ int main(int argc, char* argv[]) {
     {
         source_files = args["file"].as<std::vector<std::string> >();
     }
+//    source_files.push_back("scripts/MCMC_dating_ex4.Rev");
     
-    if ( args.count("args") and args.count("cmd"))
+    if ( args.count("args") && args.count("cmd"))
```

Comment:
This doesn't seem necessary, and is slightly less performant (probably).

Reply:
Not sure about the performance, but I thought it might be nicer to use the same coding style throughout the codebase (although I might fighting a lost cause here).